examples/khr-3hv/khr-3hv_continuous/controllers/supervisorsManager/khr3hvEnv.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+from scipy.ndimage.interpolation import shift
+
+from RobotUtils import RobotFunc
+from deepbots.supervisor.controllers.robot_supervisor import RobotSupervisor
+from gym.spaces import Box
+
+import wandb
+
+GLOBAL_DISTANCE = 0
+
+
+class khr3hvRobotSupervisor(RobotSupervisor):
```

Comment:
This caused a PEP8 warning about CamelCase class names
```suggestion
class KHR3HVRobotSupervisor(RobotSupervisor):
```